app/views/tasks/_tasks.html.slim

```diff
@@ -64,7 +64,7 @@
               = t('tasks.defaults.no_labels')
           - else
             - exercise.labels.each do |label|
-              .exercise_label style=("background-color: #{'#' + label.color.to_s}; color: #{'#' + label.font_color.to_s};")
+              .task_label style=("background-color: #{'#' + label.color.to_s}; color: #{'#' + label.font_color.to_s};")
```

Comment:
You are right, it is commented out. I renamed the CSS class from exercise_label to task_label and changed all occurrences including this one so it would not be broken if we decide to show the labels on the search page again.

Reply:
I am fine with showing the labels on the search page again 👍. When doing so, there are also a few occurrences of `exericse` that needs to be fixed 😉.

Reply:
Move to PR #912